tendrl/common/tests/test_config.py

```diff
@@ -8,11 +8,9 @@ def setup_method(self, method):
         config.os.path.exists = MagicMock(return_value=True)
 
     def test_Tendrlconfig(self, monkeypatch):
-        config.TendrlConfig()
+        config.TendrlConfig("module", "/etc/tendrl/tendrl.conf")
```

Comment:
Its not actually going to look for the file name passed here. test is just about making sure functions are called with passed arguments.